Lib/_pyrepl/reader.py

```diff
@@ -491,7 +491,7 @@ def get_prompt(self, lineno: int, cursor_on_line: bool) -> str:
             prompt = self.ps1
 
         if self.can_colorize:
-            t = THEME()
+            t = THEME(force_color=self.can_colorize)
```

Comment:
I don't think this is a good idea. The comment for `get_theme` says:

> The environment (including environment variable state and console configuration
> on Windows) can also change in the course of the application life cycle.

Here, we *force* always colorization. We should fix it differently here.

Reply:
also think about it
maybe find a way pass is mac to it
will do it later